signalfx_lambda/tracing.py

```diff
@@ -0,0 +1,70 @@
+import functools
+import os
+import opentracing
+import pip
+from jaeger_client import Config
+
+from . import utils
+
+span_prefix = ''
+
+def wrapper(func):
+    @functools.wraps(func)
+    def call(*args, **kwargs):
+        context = args[1]
+
+        tracer = init_jaeger_tracer(context)
+
+        try:
+            from signalfx_tracing import auto_instrument
+            auto_instrument(tracer)
```

Comment:
This is probably not the correct pattern for this.  It looks like the tracer is being created for each function invocation, but auto_instrument needs to happen once in an application context before any dependencies are imported.  There is currently no way to update the tracer instance without using a proxy and updating the target.

A potential workaround would be to initialize the tracer once and call a flush() wrapper at function exit instead of close().  Unfortunately we'd need to add one that will work with the current queuing process (marking tasks completed).

Reply:
I think I'll remove this for now. Will address this in a separate story